### PR TITLE
docs/meta/schema_style_guide.rst: Update description field guidance

### PR DESCRIPTION
**Pull request by duncandewhurst**
From https://github.com/open-contracting-extensions/ocds_sustainability_extension/pull/3#discussion_r1165838156

I also updated the examples with the field descriptions from the `1.2-dev` branch.

### CODE REVIEW

File: docs/meta/schema_style_guide.rst [hunk 1]
```diff
@@ -296,16 +296,16 @@ For the ``description`` field of an object:
...
 
-   A description of this tender. Structured information should be provided in the items array. Descriptions should be short and easy to read. Avoid using ALL CAPS.
+   A summary description of the tender. This complements any structured information provided using the items array. Descriptions should be short and easy to read. Avoid using ALL CAPS.
 
 ..
 
-   A description of this document. Descriptions should not exceed 250 words. In the event the document is not accessible online, the description field may be used to describe arrangements for obtaining a copy of the document.
-
+   A short description of the document. Descriptions are recommended to not exceed 250 words. In the event the document is not accessible online, the description field can be used to describe arrangements for obtaining a copy of the document.
+
```
Comment (jpmckinney) [inline docs/meta/schema_style_guide.rst:308]: Extra spaces added.

File: docs/meta/schema_style_guide.rst [hunk 2]
```diff
@@ -296,16 +296,16 @@ For the ``description`` field of an object:
 
 .. code-block:: none
 
-   A description of this <object_name>. Structured information should be provided in <related_fields>.
+   A description of this <object_name>.
 
 **Examples:**
 
-   A description of this tender. Structured information should be provided in the items array. Descriptions should be short and easy to read. Avoid using ALL CAPS.
+   A summary description of the tender. This complements any structured information provided using the items array. Descriptions should be short and easy to read. Avoid using ALL CAPS.
```
Comment (jpmckinney) [inline docs/meta/schema_style_guide.rst:303]: The second sentence is not really a desirable example to follow. We can either remove this example or find another one.

File: docs/meta/schema_style_guide.rst (hunk 1 above)
Comment (duncandewhurst) [reply docs/meta/schema_style_guide.rst]: Ah, something to do with using the web editor. Fixed in https://github.com/open-contracting/standard-development-handbook/pull/277/commits/be64c62f078213b3ce1199c49ce59936eba7db0d

File: docs/meta/schema_style_guide.rst (hunk 2 above)
Comment (duncandewhurst) [reply docs/meta/schema_style_guide.rst:303]: Given that the template is now simply 'A description of this <object_name>', I'm not sure either example is needed. What do you think?

Comment (jpmckinney) [reply docs/meta/schema_style_guide.rst:303]: I think the examples demonstrate some of the customization that can be made, though I think in most cases we will just use the template. I'm fine to keep them.

Comment (duncandewhurst) [reply docs/meta/schema_style_guide.rst:303]: Okeydokey. Assuming that you still want to replace this one with a more desirable example, how about `Bid.description`:

> A description of this bid. The description field may be used to describe the terms and conditions for the bid.

Comment (jpmckinney) [reply docs/meta/schema_style_guide.rst:303]: Hmm, I'd probably rewrite it as:

"A description of the bid, which may include terms and conditions."

Though to be honest I don't know what is meant by terms and conditions in the context of a bid.

Maybe just leave the one document example.

**Comment by jpmckinney** [comment]
I made a commit that effectively does this PR: 7e3b9b9

**Pull request closed without merging**